src/content/api/loyalty-programs.mdx

```diff
@@ -98,3 +146,30 @@ A loyalty program is a container for one or more [Promotions](/api/promotions),
 
   Returns a [paginated](/api/pagination/) list of [Loyalty Programs](#loyalty-program-model) for an Account.
 </Endpoint>
+
+---
+
+<Endpoint
+  path="/api/loyalty-programs/{loyaltyProgramId}/memberships"
+  filename="loyalty-program-memberships-create"
+>
+  ## Create Loyalty Program Memberships <Badge type="experimental"/>
+
+  This endpoint allows you to create [ Loyalty Program Memberships ](#loyalty-program-membership-model) to onboard users to [ Loyalty Programs ](#loyalty-program-model).
+
```

Comment:
Or this could change to:

`This endpoint onboards users to Loyalty Program`

Reply:
I like what you've got already :)